source/references.adoc

```diff
@@ -1,19 +1,26 @@
 [appendix]
 = References
 
-Below are references to various topics of interest around RPMs, RPM Packaging,
-and RPM Building. Some of these will be advanced and extend far beyond the
-introductory material included in this guide.
+Below are references to various topics of interest around RPMs, RPM Packaging, and RPM Building. Some of these will be advanced and extend far beyond the introductory material included in this guide.
+
+ifdef::rhel[]
+
+*   https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections] - The Red Hat Software Collections provides continuously updated development tools in latest stable versions.
```

Comment:
link:https://developers.redhat.com/products/softwarecollections/overview/[Red Hat Software Collections] - The Red Hat Software Collections offering provides continuously updated development tools in latest stable versions.